schema/data/portal/application.sql

```diff
@@ -3,7 +3,7 @@ begin;
 with rows as (
 insert into ggircs_portal.application(id, facility_id)
 overriding system value
-values (1,3)
+values (1,3), (2,2)
```

Comment:
We need to add some comments in these files, make explicit what scenarios/UIs the rows we're inserting are here for.

Reply:
I added a note in youtrack